src/utils/validateInputWithRegex.ts

```diff
@@ -10,8 +12,7 @@ export const validateAndCleanRegexInput = (
   const id = inputId.toUpperCase();
 
   const isValid = validate(id, idRegex);
-  if (!isValid)
-    throw new Error("Please check that the ID is in the correct format");
+  if (!isValid) throw new Error(i18n.t("thrownErrors.checkIdFormat"));
```

Comment:
I traced the usages of this function and found that we are catching this error and showing error alert. So this is not thrown to error boundary. In that case if we translate this, error alert side will not be able to properly identify this error by message. So we shouldn't be translating this here